### ISSUE TITLE
One-hot or multi-hot-encoding? 

### ISSUE THREAD

**Issue by sisrfeng** [shenyunhang/NA-fWebSOD#5]
Hi, I find that in your dataset flickr-clean, each image contains only 1 category. In flickr-voc, some images contain 2 categories or more. In the latter case, should the label be multi-hot? For example, an image has two annotations [category_id:0] and [category_id:2]. Should the label be [1,0,1,……,0]?
![image](https://user-images.githubusercontent.com/53520949/96249597-02437b80-0fe0-11eb-8c15-f6473208a497.png)

ref:
如果一个样本属于多个类，比如在CelebA数据集中一张人脸照片是椭圆形的，戴眼镜的... 这个时候样本的label标注是multi-hot-encoding的，也就是会出现多个标签1。如[0,1,1,0]， 1代表属性出现，0代表属性不出现。
https://www.jianshu.com/p/09c49f74308a
https://stats.stackexchange.com/questions/467633/what-exactly-is-multi-hot-encoding-and-how-is-it-different-from-one-hot

**Comment by sisrfeng** [comment]
In flickr-clean, each image contains only 1 category. Is this what you expect or just a special case? 
If the training data just contain a single object, will it lack enough feature to train a powerful object detector?

**Comment by shenyunhang** [comment]
> Hi, I find that in your dataset flickr-clean, each image contains only 1 category. In flickr-voc, some images contain 2 categories or more. In the latter case, should the label be multi-hot? For example, an image has two annotations [category_id:0] and [category_id:2]. Should the label be [1,0,1,……,0]?

Yes, you are right. The paper uses one-hot labels to expose our method and it is easy to generalize to multi-hot labels.

> In flickr-clean, each image contains only 1 category. Is this what you expect or just a special case?

flickr-clean is not created by us.
I think it is expected.

> If the training data just contain a single object, will it lack enough feature to train a powerful object detector?

I agree with this.
I think it will lack the images with multi-class co-occurrence.
